.github/workflows/black.yml

```diff
@@ -8,6 +8,5 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - uses: actions/setup-python@v2
-    - uses: psf/black@stable
-      with:
-        version: "22.1.0"
+    - name: Check code with black
+      run: touch .env && docker-compose run black
```

Comment:
I don't understand this change with respect to the previous behavior. Could you explain what that is doing?

Reply:
Is this because you are setting the black version in the requirements-dev, so it's no longer needed here?

Reply:
Yes, this will build the Docker image with the black version specified in `dev-requirements.txt` and then create a container from that image that then checks the code with black.

Reply:
The benefit is that most of the logic is moved from the workflow to Docker which you can also run locally.